Update RunConfiguration with TestCaseFilter info (#9382)
MicrosoftDocs/visualstudio-docs: docs/test/configure-unit-tests-by-using-a-dot-runsettings-file.md

|**TargetPlatform**|x86|**Omit this whole tag to auto-detect.**<br /><br />This setting defines the architecture to use to run tests. Possible values are `x86`, `x64`, `ARM`, `ARM64`, `S390x`.<br /><br />When autodetecting, the architecture for AnyCPU DLLs may differ based on the runner. For .NET Framework runner (in Visual Studio, or vstest.console.exe in Developer command line), the default is x86. For .NET runner (dotnet test), the default is the current process architecture.<br /><br />|
|**TreatTestAdapterErrorsAsWarnings**|false|false, true|
|**TestAdaptersPaths**||One or more paths to the directory where the TestAdapters are located|

> |**TestCaseFilter**|| A filter expression in the format __\<property>\<operator>\<value>[\|\&amp;\<Expression>]__. Pay attention to the boolean operator **\&** that should be used as a HTML entity **\&amp;**. Expressions can be enclosed in paranthesis. For detailed syntax on expression structure read [vstest-docs](https://github.com/microsoft/vstest-docs/blob/main/docs/filter.md). |

Reviewer: @Mikejo5000 is it better to reuse the exact same description as this one https://github.com/MicrosoftDocs/visualstudio-docs/blob/main/docs/test/vstest-console-options.md:

![image](https://github.com/MicrosoftDocs/visualstudio-docs/assets/11340282/1ef63318-9365-4aa1-9948-c89d6b0b214b)

Author: The description above, in addition to some typos, only shows the boolean "or" operator. The filter expression can be built with any of the logical operators ("and", "or"). It is important to mention that the "and" operator must be expressed as an HTML element.

Reviewer: @matei-tm that makes sense. Would you mind opening a PR on the other page to update the description?

> |**TestCaseFilter**|| A filter expression in the format __\<property>\<operator>\<value>[\|\&amp;\<Expression>]__. Pay attention to the boolean operator **\&** that should be used as a HTML entity **\&amp;**. Expressions can be enclosed in paranthesis. For detailed syntax on expression structure read [vstest/docs/filter.md](https://github.com/microsoft/vstest/blob/main/docs/filter.md). |

Reviewer 2: ```suggestion
|**TestCaseFilter**|| A filter expression in the format \<property>\<operator>\<value>[\|\&amp;\<Expression>]. The boolean operator **\&** should be represented by the HTML entity **\&amp;**. Expressions can be enclosed in parentheses. For detailed syntax on expression structure, see [vstest/docs/filter.md](https://github.com/microsoft/vstest/blob/main/docs/filter.md). |
```

Reviewer 2: I removed underscores for bold, as we don't use this syntax.
